web/srv/server.go

```diff
@@ -111,6 +111,7 @@ func NewServer(addr, templateDir, staticDir, uuid, controllerNamespace, webpackD
 	server.router.GET("/api/services", handler.handleApiServices)
 	server.router.GET("/api/tap", handler.handleApiTap)
 	server.router.GET("/api/routes", handler.handleApiTopRoutes)
+	server.router.GET("/profiles/new", handler.handleProfileDownload)
```

Comment:
Would you mind moving this to the "webapp routes" section above? I don't think it belongs here in the "webapp api routes" section, since it's not a /api route.